cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -1213,6 +1214,26 @@ func checkDataAndSavePairing(rootdir string) (bool, error) {
 	return subtle.ConstantTimeCompare(marker1, marker2) == 1, nil
 }
 
+var (
+	// can be mocked in tests
+	partSrcPollIterations = 120
+	partSrcPollWait       = 500 * time.Millisecond
```

Comment:
Given that the poll is very inexpensive, I'd do it more often, maybe every 50ms even.

Reply:
Changed